Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonDataGridViewCheckBoxCell.cs

```diff
@@ -303,7 +303,7 @@ private Point MouseEnteredCellAddressInternal
 
                 // Grab the internal property implemented by base class
                 // ReSharper disable RedundantBaseQualifier
-                return (Point)_piMouseEnteredCellAddress.GetValue(base.DataGridView, null);
+                return (Point)_piMouseEnteredCellAddress?.GetValue(base.DataGridView, null)!;
```

Comment:
Should these be using `as` like the other changes above ?

Reply:
Used `is` & other ReSharper suggestions